### PR TITLE
Doc changes for Options and constructor warning.

### PR DESCRIPTION
**Pull request by rdepena**
Doc changes with the following:
V1 style Application options on the create `create` function
V1 style Window options on the create `create` function

"*DO NOT USE!* Use `create`" warning on Application/Window constructors.

### CODE REVIEW

**Comment by pbaize** [comment]
Should these changes wait for the application start/quit changes and reverence start instead of create?

File: src/api/application/application.ts [hunk 1]
```diff
@@ -128,6 +188,9 @@ export class Application extends EmitterBase<ApplicationEvents> {
     public _manifestUrl?: string;
     private window: _Window;
 
+    /**
+     * **DO NOT USE!** Please use {@link Application.create Application.create}
+     */
```
Comment (licui3936) [inline src/api/application/application.ts]: This could be removed. It's handled in event doc.

File: src/api/window/window.ts [hunk 2]
```diff
@@ -521,6 +719,10 @@ export class _Window extends EmitterBase<WindowEvents> {
      "load-failed", "failed", "succeeded"
      */
 
+    /**
+     * **DO NOT USE!**
+     * Please use {@link Window.create Window.create}
+     */
```
Comment (licui3936) [inline src/api/window/window.ts]: same here

**Comment by rdepena** [comment]
@pbaize I can change once we consume, easy either way.

**Pull request merged**